frontend/src/Components/StoreSearch/StoreSearch.js

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import StoreCard from "./StoreCard";
+import {Container, Col, Row, Jumbotron, Button, ButtonGroup, Card, InputGroup, FormControl} from "react-bootstrap";
+import './StoreSearch.css'
+
+const stores = [
+    {
+        name: "Super Market",
+        address: "123 E Green St. Champaign, IL",
+        hours: [9, 18],
+        distance: 6
+    },
+    {
+        name: "Local Grocer",
+        address: "456 Testing Ave. Boston, MA",
+        hours: [8, 16],
+        distance: 11
+    }
+    ]
+
+const getLocation = () => {
+    if (navigator.geolocation) {
+        navigator.geolocation.getCurrentPosition(processLocation, locationError);
+    } else {
+        console.error("Geolocation is not supported by this browser.");
+    }
+};
+
+const processLocation = (pos) => {
+    console.log([pos.coords.latitude, pos.coords.longitude]);
+};
+
+const locationError = (error) => {
+    switch(error.code) {
+        case error.PERMISSION_DENIED:
```

Comment:
should the case be `PERMISSION_DENIED` rather than `error.PERMISSION_DENIED`? Not to sure, if you tested this and it works then ignore this comment :)

Reply:
I tested this and it works, but in short `error.code` is getting the actual error type from the error object and anything in snake case (e.g. `error.PERMISSION_DENIED`) is a static value always contained in the error object for comparisons.